website/test/unit/store/users.test.js

```diff
@@ -29,15 +29,13 @@ describe('Store/users', () => {
 
   describe('action/fetchUsers', () => {
     let commit
-    
     beforeEach(() => {
       commit = spy()
     })
 
     afterEach(() => {
       restore()
     })
-    
```

Comment:
@pallabez This blank line is not needed.

Reply:
Got it